### ISSUE TITLE
Center loss didn't update in code

### ISSUE THREAD

**Issue by harrychangdev** [davidsandberg/facenet#673]
There is a center_loss function in facenet.py
it will return loss and centers
when execute loss op can calculate center loss
and execute centers op can update the center

But!!!!
in train_softmax.py
`prelogits_center_loss, _ = facenet.center_loss(prelogits, label_batch, args.center_loss_alfa, nrof_classes)
tf.add_to_collection(tf.GraphKeys.REGULARIZATION_LOSSES, prelogits_center_loss * args.center_loss_factor)`
the code omit the centers op which center_loss function return
this code don't care about center update and center will always the same

i think this is a bug right? or has something I misunderstanding?

**Comment by davidsandberg** [comment]
It's a bug and has been fixed in efc2f94dd1c6806aa249ba418aa118ce6f1165df.
